tests/Feature/MovieEditTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Movie;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieEditTest extends TestCase
+{
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_movie_edit_screen_can_render()
+    {
+        $user = User::factory()->create();
+        $movie = Movie::factory()->create();
+
+        $this->actingAs($user)
+        ->get('/movies/'.$movie->id.'/edit')
+        ->assertStatus(200);
+    }
+
+    public function test_movie_edit_guest_cannot_access()
+    {
+        $movie = Movie::factory()->make();
+
+        $this->get('/movies/'.$movie->id.'/edit')
+        ->assertStatus(404);
```

Comment:
This one I believe it should redirect to login no? The movie exists so it does not make sense that it is returning a 404. 
In fact, asserting that the user cannot edit an unexisting movie can be a test case as well.

Reply:
No exactly sure how I should conduct test this;

The test I wanted to do was a fail; where if the user isn't logged in it should no allow access to edit

My assumption was since a user needs to be logged in to access edit, any test that doesn't have `actingAs() ` in it would fail

Though it probably would be better to test that it redirects since that is how the program functions

Again though I am no exactly sure how to test this

Reply:
Thanks for your clarification on slack. Manage to get the test functional